README.md

```diff
@@ -123,15 +123,15 @@ Google Cloud BigQuery (Alpha)
 
 Here is a code snippet showing a simple usage example from within Compute/App Engine. Note that you
 must [supply credentials](#authentication) and a project ID if running this snippet elsewhere.
+Complete source code can be found at
+[gcloud-java-examples:com.google.gcloud.examples.bigquery.snippets.CreateTableAndLoadData](https://github.com/GoogleCloudPlatform/gcloud-java/tree/master/gcloud-java-examples/src/main/java/com/google/gcloud/examples/bigquery/snippets/CreateTableAndLoadData.java).
```

Comment:
That would be my preference.

* Complete source code can be found at the <ins>CreateTableAndLoadData.java</ins> snippet.
* Complete source can be found <ins>here</ins>.
* For the complete source see <ins>CreateTableAndLoadData.java</ins>.
* The complete snippet is located at <ins>CreateTableAndLoadData.java<ins>.